services/src/main/java/org/keycloak/services/resources/account/AccountRestService.java

```diff
@@ -395,17 +395,6 @@ public Response updateConsent(final @PathParam("clientId") String clientId,
                                   final ConsentRepresentation consent) {
         return upsert(clientId, consent);
     }
-    
-    @Path("/totp/remove")
-    @DELETE
-    public Response removeTOTP() {
-        auth.require(AccountRoles.MANAGE_ACCOUNT);
-        
-        session.userCredentialManager().disableCredentialType(realm, user, CredentialModel.OTP);
```

Comment:
Just a very small nit pick.
```java
import org.keycloak.credential.CredentialModel;
```
is also unused now.

Reply:
Easy to fix, so I removed it :-)